src/routes/professors/[name]/+page.server.ts

```diff
@@ -77,16 +57,11 @@ export const load: PageServerLoad = async ({ params }) => {
 	const ratings = await db
 		.select()
 		.from(ratingsTable)
-		.where(
-			inArray(
-				ratingsTable.courseId,
-				courses.map((course) => course.id)
-			)
-		);
+		.where(eq(ratingsTable.professorId, professor[0].id));
 
-	const extendedCourses: ExtendedCourse[] = courses.map((course) => ({
+	const extendedCourses = courses.map((course) => ({
 		...course,
-		averageRating: undefined
+		averageRating: 0
```

Comment:
We still want it to be undefined, we just need to assert that its a number
```suggestion
		averageRating: undefined as number | undefined
```